tests/test_switch.py

```
@@ -1,10 +1,12 @@
 """Test openevse switches."""
 
 import logging
-from unittest.mock import MagicMock, patch
+from asyncio import TimeoutError
```

Ruff (0.15.10)

[error] Import `TimeoutError` is shadowing a Python builtin

(A004)